### DISCUSSION TITLE
请问这个是哪里的问题呀

### DISCUSSION THREAD

**Discussion by CHENGZI9621** [wangxinleo/wechat-public-account-push#56] [General]
![1661850078244](https://user-images.githubusercontent.com/112463401/187396435-0906f410-4059-4422-8586-53b82f214b5a.png)
![1661850099090](https://user-images.githubusercontent.com/112463401/187396459-25711d7f-dc94-4077-a40d-57da1d3055d9.png)

**Comment by wangxinleo** [comment]
可能入参不对，建议看看Q&A中关于此部分的对应关系，详细检查一下看看。
